build/common.ps1

```diff
@@ -492,10 +573,23 @@ Function Invoke-ILMerge {
         [string]$Configuration = $DefaultConfiguration
     )
     $nugetArtifactsFolder = Join-Path $Artifacts "$Configuration\NuGet.Clients\NuGet.CommandLine"
-    pushd $nugetArtifactsFolder
+    $dlls = Get-ChildItem $nugetArtifactsFolder -Filter 'nuget.*.dll' | %{ $_.Name }
```

Comment:
This is nice, but aren't some dlls like xplat getting in now? The CI is I think using a different script for this, so for the official build they are likely excluded and this can still work.